### PR TITLE
feat(cli): enable to print the config schema non merged

### PR DESCRIPTION
**Pull request by djamaile**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/cli/src/commands/index.ts [hunk 1]
```diff
@@ -365,6 +365,7 @@ export function registerCommands(program: Command) {
       '--format <format>',
       'Format to print the schema in, either json or yaml [yaml]',
     )
+    .option('--non-merged', 'Enable to print the config schema non merged')
```
Comment (freben) [inline packages/cli/src/commands/index.ts:368]: Enabling/disabling flags are commonly named `--foo` / `--no-foo`.

https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue

This one should be named `--no-merge`, and then you read `opts.merge`

Comment (djamaile) [reply packages/cli/src/commands/index.ts]: Gotcha, updated it

File: packages/cli/src/commands/config/schema.ts [hunk 2]
```diff
@@ -28,19 +28,23 @@ export default async (opts: OptionValues) => {
...
-    (schema.serialize().schemas as JsonObject[]).map(
-      _ => _.value as JSONSchema,
-    ),
-  );
-
-  merged.title = 'Application Configuration Schema';
-  merged.description =
-    'This is the schema describing the structure of the app-config.yaml configuration file.';
+  let configSchema: JsonObject | JSONSchema;
+  if (opts.merged) {
```
Comment (UsainBloot) [inline packages/cli/src/commands/config/schema.ts:32]:
```suggestion
  if (opts.merge) {
```

This might read a bit nicer with `--no-merge and --merge`

Comment (djamaile) [reply packages/cli/src/commands/config/schema.ts:32]: Alright lets go for it

**Review by freben — APPROVED** [review]
Aight, all good!

**Pull request merged**